Src/WitsmlExplorer.Frontend/components/Modals/AnalyzeGapModal.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import { NavigationAction } from "../../contexts/navigationAction";
+import { DisplayModalAction, HideModalAction } from "../../contexts/operationStateReducer";
+import LogObject from "../../models/logObject";
+import JobService, { JobType } from "../../services/jobService";
+import { WITSML_INDEX_TYPE_DATE_TIME, WITSML_INDEX_TYPE_MD } from "../Constants";
+import ModalDialog from "./ModalDialog";
+import { TextField } from "@material-ui/core";
```

Comment:
Use TextField from "@equinor/eds-core-react" instead of material-ui

Reply:
fixed